curriculum/challenges/english/01-responsive-web-design/css-picasso-painting/part-033.md

```diff
@@ -0,0 +1,134 @@
+---
+id: 60b69a66b6ddb80858c51597
+title: Part 33
+challengeType: 0
+dashedName: part-33
+---
+
+# --description--
+
+And give the table a `z-index` of `1`.
```

Comment:
**Personal Preference**: Instructions/descriptions which start as though they are continuing from the previous lesson never read right to me 🤷‍♂️ 
```suggestion
Give the table a `z-index` of `1`.
```